res/values/strings.xml

```diff
@@ -43,6 +43,7 @@
         <item>Private Seafile Server</item>
         <item>SeaCloud.cc</item>
         <item>cloud.seafile.com</item>
+        <item>@string/shib_login_title</item>
```

Comment:
Have you tested it under a non-English lanugage? For example, with the phone lanugage set to Chinsese, When you click "add an account", "Shibboleth Login" still won't show, since it has has no correspondent item in Chinese  translation yet.